--- ./security-admin/src/main/java/org/apache/ranger/service/XAccessAuditService.java	2018-07-05 12:30:06.000000000 -0500
+++ /tmp/XAccessAuditService.java	2018-08-27 04:56:23.072116219 -0500
@@ -121,6 +121,7 @@
 	}
 
 	protected VXAccessAudit mapEntityToViewBean(VXAccessAudit vObj, XXAccessAudit mObj) {
+		vObj.setId( mObj.getId());
 		vObj.setAuditType( mObj.getAuditType());
 		vObj.setAccessResult( mObj.getAccessResult());
 		vObj.setAccessType( mObj.getAccessType());
@@ -141,7 +142,7 @@
 		vObj.setSequenceNumber( mObj.getSequenceNumber());
 		vObj.setEventCount( mObj.getEventCount());
 		vObj.setEventDuration( mObj.getEventDuration());
-		vObj.setTags(mObj.getTags());
+		vObj.setAction( mObj.getAction());
 
 		XXService xService = daoManager.getXXService().findByName(mObj.getRepoName());
 		if (xService != null) {
